tasks/elasticsearch-pipeline.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: ensure pipelines dir is created
+  file:
+    path: /etc/elasticsearch/pipelines
+    state: directory
+    owner: "{{ es_user }}"
+    group: "{{ es_group }}"
+
+- name: Copy pipelines to elasticsearch
+  copy: src={{ item }} dest=/etc/elasticsearch/pipelines owner={{ es_user }} group={{ es_group }}
+  register: load_pipelines
+  with_fileglob:
+    - "{{ es_pipelines_fileglob | default('') }}"
```

Comment:
What does the `default('')` do here? This variable is defined in defaults so it will always be set.